Orange/widgets/data/owcolor.py

```diff
@@ -275,14 +275,10 @@ def change_color(self, index):
 
 
 class OWColor(widget.OWWidget):
-    """
-    Set color legend for variables
-
-    Widget for defining specific color palettes for discrete and continuous
-    features
-    """
-
     name = "Color"
+    description = "Set color legend for variables."
+    long_description = "Widget for defining specific color palettes " \
```

Comment:
Fixed. I hope it's correct. Can you check especially description.py and then merge?